### ISSUE TITLE
Video layout broken due to random layout update

### ISSUE THREAD

**Issue by GenericCherrySwitch** [omnidevZero/YouTubeRedux#76]
The video page's layout had recently changed which caused parts of it to break like the like and view count, channel-related buttons, etc.
![Screenshot_1411](https://user-images.githubusercontent.com/44690814/196304330-de2e2514-81a3-4043-9a8a-2ee253717c23.png)

**Comment by gameshowrecaps** [comment]
Yep. I kept going back and forth in the experiments and YouTube gave me the new layout yet again. This layout is so terrible.

**Comment by KingGamingYT** [comment]
Duplicate of #64 and #69.

**Comment by lynrayy** [comment]
So, what is next? What did you mean by this? Noone care
![image](https://user-images.githubusercontent.com/59417884/196596972-f33f59d7-5988-4e41-8dfe-bbe0cc59c7ae.png)

**Comment by KingGamingYT** [comment]
> So, what is next? What did you mean by this? Noone care
> ![image](https://user-images.githubusercontent.com/59417884/196596972-f33f59d7-5988-4e41-8dfe-bbe0cc59c7ae.png)

It means this issue is the same, AKA a duplicate, as the situation and problem presented in those two issues, making this one redundant.
